tests/conform-yup.spec.ts

```diff
@@ -1,41 +1,65 @@
 import { describe, test, expect } from 'vitest';
 import { parseWithYup, getYupConstraint } from '@conform-to/yup';
 import * as yup from 'yup';
-import { STATE } from '@conform-to/dom';
+import { type Constraint, STATE } from '@conform-to/dom';
 import { createFormData } from './helpers';
 
+// Adding custom method tuple until Yup is updated to v1 where the tuple method is built-in
+// ref: https://github.com/jquense/yup/issues/528#issuecomment-916885944
+yup.addMethod(yup.array, 'tuple', function (schema) {
+	if (!this.isType(schema)) yup.ValidationError();
+	return yup.object({
+		...Object.fromEntries(Object.entries(schema)),
+	});
+});
+
 describe('conform-yup', () => {
+	const maxDate = new Date();
 	const schema = yup
 		.object({
 			text: yup
 				.string()
+				.required('required')
 				.min(1, 'min')
 				.max(100, 'max')
 				.matches(/^[A-Z]{1-100}$/, 'regex'),
 			tag: yup.string().required('required').oneOf(['x', 'y', 'z'], 'invalid'),
 			number: yup.number().required().min(1, 'min').max(10, 'max'),
 			timestamp: yup
 				.date()
-				.min(new Date(1), 'min')
-				.max(new Date(), 'max')
+				.required('required')
+				.min(new Date(1).toISOString(), 'min')
+				.max(maxDate.toISOString(), 'max')
 				.default(new Date()),
 			options: yup
-				.array(yup.string().oneOf(['a', 'b', 'c'], 'invalid'))
+				.array(
+					yup.string().required('required').oneOf(['a', 'b', 'c'], 'invalid'),
+				)
+				.required('required')
 				.min(3, 'min'),
 			nested: yup
 				.object({
 					key: yup.string().required('required'),
 				})
-				.test('nested', 'error', () => false),
```

Comment:
Can you keep this check? I wanna make sure group level fields are still validated properly.